INFO     root:test_calculator.py:135 Starting test for valid subtraction operation with login.
INFO     root:test_calculator.py:142 Attempting to log in with credentials: {'userName': 'admin', 'password': '123'}
INFO     root:test_calculator.py:144 Login response status code: 200
INFO     root:test_calculator.py:147 Login response data: {'message': None, 'data': {'success': True, 'token': '60bf8882-40c1-4f9c-97fa-dd41ff66b2bb'}, 'status': 'success'}
INFO     root:test_calculator.py:149 Login response data: {'message': None, 'data': {'success': True, 'token': '60bf8882-40c1-4f9c-97fa-dd41ff66b2bb'}, 'status': 'success'}
INFO     root:test_calculator.py:151 Received token: 60bf8882-40c1-4f9c-97fa-dd41ff66b2bb
INFO     root:test_calculator.py:159 Sending calculation request: {'op1': -5, 'operation': '-', 'op2': -8}
INFO     root:test_calculator.py:164 Received response with status code: 200
INFO     root:test_calculator.py:167 Calculation response data: {'message': None, 'data': 3, 'status': 'success'}
INFO     root:test_calculator.py:173 The result of the addition operation is correct.